ast.go

```diff
@@ -127,6 +151,11 @@ func (g *graph) hydrateFile(pkg Package, f *descriptor.FileDescriptorProto) File
 	}
 	g.add(fl)
 
+	for _, dep := range f.GetDependency() {
+		fileDep := g.mustSeen(dep).(File)
```

Comment:
From what I understand, this is going to true only because the `req.GetProtoFile()` returns the files in the topological order, and hence we would have seen an imported proto file before processing the file itself. 
Should we have a comment here to explain this?

Reply:
sure i'll add a comment